benefits/core/views.py

```diff
@@ -12,7 +12,7 @@
 
 def PageTemplateResponse(request, page_vm):
     """Helper returns a TemplateResponse using the common page template."""
-    return TemplateResponse(request, "core/page.html", page_vm.context_dict())
+    return TemplateResponse(request, "core/index.html", page_vm.context_dict())
```

Comment:
`PageTemplateResponse` is used in many places, so you are potentially changing a lot more than you should for the effect you are trying to get (to make the `index` function use the `core/index.html` template).

The return statement of [`index`](https://github.com/cal-itp/benefits/pull/406/files#diff-2e5e461b0d86b207a87071f948630540090d2d020e2b0e053b5486e6ac5c82fdR51) is what you should be changing. Feel free to ask me any questions if you feel unsure on this!

Reply:
I didn't notice this, thanks for catching!